src/HDInsight/HDInsight.Test/ScenarioTests/TestController.cs

```diff
@@ -46,6 +47,9 @@ public class TestController : RMTestBase
         public KeyVaultClient KeyVaultClient { get; private set; }
         public ManagedServiceIdentityClient ManagedServiceIdentityClient { get; private set; }
         public NetworkManagementClient NetworkManagementClient { get; private set; }
+
+        public GraphRbacManagementClient GraphRbacManagementClient { get; private set; }
```

Comment:
```suggestion
        public GraphRbacManagementClient GraphRbacManagementClient { get; private set; }
```
`GraphRbacManagementClient` is deprecating. Please use [MicrosoftGraphClient](https://github.com/Azure/azure-powershell-common/blob/main/src/Graph.Rbac/MicrosoftGraph/Version1_0/MicrosoftGraphClient.cs) instead

Reply:
Fixed